benchmarks/vericoding_dafny/template.yaml

```diff
@@ -0,0 +1,25 @@
+vc-description: |-
```

Comment:
I would put in comments telling the ai what each of these keys is for, and an order may be warranted.